e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -52,4 +64,26 @@ describe('Upload Files', () => {
             });
         });
     });
+
+    it('MM-T12 Loading indicator when posting images', () => {
+        const filename = 'huge-image.jpg';
```

Comment:
I think this needs to visit again the channel since the previous test left the RHS open.

Reply:
I will change `before` to `beforeEach`, so the test case will be more predictable.